setup_ch4_inversion.sh

```diff
@@ -202,7 +203,7 @@ if "$CreateStateVectorFile"; then
     printf "\n=== CREATING STATE VECTOR FILE ===\n"
     
     # Use GEOS-FP or MERRA-2 CN file to determine ocean/land grid boxes
-    LandCoverFile="${DataPath}/GEOS_${gridDir}/${metDir}/${constYr}/01/${metUC}.${constYr}0101.CN.${gridRes}.nc"
+    LandCoverFile="${DataPath}/GEOS_${gridDir}_NA/${metDir}/${constYr}/01/${metUC}.${constYr}0101.CN.${gridRes}.NA.nc"
```

Comment:
The GEOS_0.25x0.3125_NA landcover files for 2018 is already present on the ami, but the GEOS_0.25x0.3125 data for 2018 is not present -- there is only 2011 data. My assumption is that we should use the NA data, but I don't know the difference? Or if we should download the corresponding landcover files from elsewhere?

Reply:
I'm also not sure about this. Would seem OK to me to use a land-cover file for 2011, and we probably want one that can be used for any domain. Not sure if we need a different file for each year, or just select one to use as default? Using a global file for a particular year as default would be one less thing to worry about.

Reply:
Okay, then I will update this to use the 2011 land cover file instead👍

Reply:
Do we know why it's looking for a landcover file for 2018? It should only look for the constant year (`constYr`) for that met field since landcover is a constant field. For GEOS-FP, `constYr=2011` and for MERRA-2 `constYr=2015` -- both values are defined further up in `setup_ch4_inversion.sh`.

Reply:
Also, we should use the NA file by default since our example case uses a domain in North America (and NA met fields). If users choose a different domain like `CH`, `AS`, or `EU` then we should have the script automatically replace the region token in that path (I can look into that). Users may also choose to manually update this file to use a different region if they've pre-cropped met fields on their end. The global, high-resolution land cover file may also be used -- we just avoid using those files for all met fields because of the negative impact in I/O when reading/regridding those fields in HEMCO.

Reply:
Right, and `${gridDir}` include `"_NA"` if `$Region="NA"` as by default. Therefore we shouldn't need to specify `GEOS_${gridDir}_NA`

Reply:
> Also, we should use the NA file by default since our example case uses a domain in North America (and NA met fields). If users choose a different domain like `CH`, `AS`, or `EU` then we should have the script automatically replace the region token in that path (I can look into that). Users may also choose to manually update this file to use a different region if they've pre-cropped met fields on their end. The global, high-resolution land cover file may also be used -- we just avoid using those files for all met fields because of the negative impact in I/O when reading/regridding those fields in HEMCO.

Right, I haven't been able to figure out why the region token isn't being included automatically, as I alluded to above. Also, I didn't think about region-specific land-cover files. But that's totally fine, as long as we have one to go along with the met data for each of the `NA`, `CH`, `AS`, `EU` domains.

Reply:
I don't know why the Region wasn't getting picked up. I will rerun the setup script and see what is going on.